index.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+const AWS = require('aws-sdk');
+const ini = require('ini-config-parser');
+const os = require('os');
+
+const fallbackAWSRegion = 'us-east-1';
+
+// the order of precedence for setting the credentials is:
+// 1. --profile command-line parameter
+// 2. AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables
+// 3. AWS_PROFILE or default
+const initCustomCredentials = (params) => {
```

Comment:
since we aren't really doing much with aws-sdk, should we just pass it in so we can remove as a depedency?